Sources/SourceKitLSP/TestDiscovery.swift

```diff
@@ -257,15 +254,14 @@ extension SourceKitLSPServer {
 
     let syntacticTests = try await languageService.syntacticDocumentTests(for: req.textDocument.uri)
 
-    if let index = workspace.index {
+    if let index = workspace.index(checkedFor: .imMemoryModifiedFiles(documentManager)) {
```

Comment:
```suggestion
    if let index = workspace.index(checkedFor: .inMemoryModifiedFiles(documentManager)) {
```
Weirdly this is the first time I noticed the misspelling (despite there being many before this).